fix: threadList showing errors as `[object, object]` (#36004)
RocketChat/Rocket.Chat: .changeset/sour-pianos-glow.md

---
"@rocket.chat/meteor": patch
---

> fixes issue that caused errors on the threads list to be shown as `[object, Object]`

Reviewer: ```suggestion
Fixes an issue with the Thread List displaying unreadable errors.
```
